test/fixtures/output/connect/plugin/install/remote-bad-sha1.golden

```diff
@@ -0,0 +1,10 @@
+Beginning installation.
+
+Using the only available Confluent Platform installation at "test/fixtures/input/connect/confluent-7.3.3".
```

Comment:
What if the user has two versions of CP installed, but this command only detects one of them? I think we should always prompt, unless it's explicitly passed with a flag.

Reply:
The CP installation is only used to find the standard installation directory if they didn't supply `--plugin-directory` or the worker config files if they didn't supply `--worker-configs`.

So in this case, the user could just decline on the next prompt and rerun with both flags.

If it's better UX to add a flag where users can supply a CP installation, I can do that.